teuthology/task/kernel.py

```diff
@@ -1021,11 +1049,37 @@ def task(ctx, config):
             if need_to_install_distro(ctx, role):
                 need_install[role] = 'distro'
                 need_version[role] = 'distro'
+        elif role_config.get("koji", None):
+            # installing a kernel from koji
+            build_id = role_config.get("koji")
+            if system_type != "rhel":
```

Comment:
Cool.  Perhaps we should allow any rpm based systems?

On Mon, Jan 26, 2015 at 11:06 AM, Zack Cerza <notifications@github.com>
wrote:

> In teuthology/task/kernel.py
> <https://github.com/ceph/teuthology/pull/417#discussion_r23545791>:
>
> > @@ -1021,11 +1049,37 @@ def task(ctx, config):
> >              if need_to_install_distro(ctx, role):
> >                  need_install[role] = 'distro'
> >                  need_version[role] = 'distro'
> > +        elif role_config.get("koji", None):
> > +            # installing a kernel from koji
> > +            build_id = role_config.get("koji")
> > +            if system_type != "rhel":
>
> I think we'd want it to work for Fedora too.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ceph/teuthology/pull/417/files#r23545791>.
>